venezuela/info.json

```diff
@@ -6,5 +6,6 @@
   "zim_file": "content.zim",
   "embed_zim": false,
   "ic_launcher": "icon.png",
+  "ic_launcher_master": "icon_master.png",
```

Comment:
because this background is not a full color we will have to supply 2 images, 1 the background, 1 the foreground (the stars)

Reply:
done